docs/source/user-guide/tutorials/build-pkgs.rst

```diff
@@ -244,20 +243,19 @@ Replace this ``source`` section:
 
 .. code-block:: bash
 
-   git_rev: v0.13.1
-   git_url: https://github.com/joerick/pyinstrument.git
+   git_rev: v0.6.7
+   git_url: https://github.com/pallets/click.git
 
 With the following:
 
 .. code-block:: bash
 
-    fn: pyinstrument-0.13.1.tar.gz
-    md5: e347036acc50720c0903dc2221b2605d
-    url: https://pypi.python.org/packages/source/p/pyinstrument/pyinstrument-0.13.1.tar.gz
+    url: https://files.pythonhosted.org/packages/95/d9/c3336b6b5711c3ab9d1d3a80f1a3e2afeb9d8c02a7166462f6cc96570897/click-6.7.tar.gz
```

Comment:
update for click 7.0 

https://files.pythonhosted.org/packages/f8/5c/f60e9d8a1e77005f664b76ff8aeaee5bc05d0a91798afd7f53fc998dbc47/Click-7.0.tar.gz

5b94b49521f6456670fdb30cd82a4eca9412788a93fa6dd6df72c94d5a8ff2d7

Reply:
I'm not really sure keeping this up to date is a good idea.  It will fall out of date over time, and we don't really want to be maintaining our docs to keep up with them.  Updating it right now is OK, but let's not try to keep the docs always representative of exactly what users will see.  It's just not a good use of our time.